src/analyticalSolutions/sedovSolution/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+set(SOURCES FileData.cpp SedovSolution.cpp main.cpp)
+add_executable(sedovSolution ${SOURCES})
+target_include_directories(sedovSolution PRIVATE ${PROJECT_SOURCE_DIR}/include)
+target_include_directories(sedovSolution PRIVATE ${PROJECT_SOURCE_DIR}/src)
+install(TARGETS sedovSolution RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

There is a failure when building `sedovSolution` with clang++:
```
ld.lld: error: undefined symbol: std::filesystem::__cxx11::path::_M_split_cmpts()
>>> referenced by main.cpp
```

This fixes the issue (add it please):
```suggestion
target_link_libraries(sedovSolution stdc++fs)
install(TARGETS sedovSolution RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```